src/functionaltests/java/com/ericsson/ei/threadingAndWaitlistRepeat/ThreadingAndWaitlistRepeatSteps.java

```diff
@@ -46,14 +46,13 @@
 
     @Before("@ThreadingAndWaitlistRepeatScenario")
     public void beforeScenario() throws IOException {
-        File logfile = new File(LOGFILE);
-        logfile.createNewFile();
-        System.setOut(new PrintStream(logfile));
+        tempLogFile = File.createTempFile("logfile", ".tmp");
+        tempLogFile.deleteOnExit(); 
+        System.setOut(new PrintStream(tempLogFile));
     }
 
     @After("@ThreadingAndWaitlistRepeatScenario")
     public void afterScenario() throws IOException {
```

Comment:
Is this used at all? If not, it can be removed.

Reply:
It is used